src/server.js

```diff
@@ -31,8 +31,8 @@ export default async function start({
   plugins: Array<mixed>,
 }) {
   const tls = config.get('server.secure') && {
-    key: fs.readFileSync(path.resolve(__dirname, config.get('server.tlsKey'))),
-    cert: fs.readFileSync(path.resolve(__dirname, config.get('server.tlsCert'))),
+    key: fs.readFileSync(path.resolve('./', config.get('server.tlsKey'))),
```

Comment:
I'd suggest you revert this changes. `__dirname` will always resolve to the absolute path of the directory containing  the currently executing file. Whereas `.` gives you the directory where the `node` process is invoked. https://stackoverflow.com/questions/8131344/what-is-the-difference-between-dirname-and-in-node-js

To actually test your changes, you can run https://github.com/jabdul/crud-api/blob/368e2a0fd624b22a0bd03ef9e3c6474f173a6877/package.json#L9
```bash
$ yarn bootstrap
```
from the root level of this repo.

Reply:
@Ehbraheem that is the exact reason that this change was made. Have discussed this with James on Monday

Reply:
I am afraid to tell you that this still doesn't solve the problem you expect. It instead create more problem.

You can try this, make this exact changes to this lines https://github.com/jabdul/crud-api/blob/368e2a0fd624b22a0bd03ef9e3c6474f173a6877/lib/server.js#L91 https://github.com/jabdul/crud-api/blob/368e2a0fd624b22a0bd03ef9e3c6474f173a6877/lib/server.js#L92 in one of the Microservices that is dependent on `crud-api` (like say the **Elephant auth**).
`SERVICE/node_modules/@ctt/crud-api/lib/server.js`

**NB**: I did already before the **RFC**

Reply:
What problems do you expect? The reason for the change was so that we wouldn't need to hack around inside `node_modules`. Surely it shouldn't resolve from the root inside a node_module that could be n levels deep or symlinked to elsewhere?